packages/ubus-lime-utils/files/etc/udhcpc.user

```diff
@@ -0,0 +1,19 @@
+setup_hotspot_watchping() {
```

Comment:
Wow this is the first time I came to `udhcpc.user` file (https://udhcp.busybox.net/README.udhcpc), it is nice to learn from this!